src/graph_compute/update_function.h

```diff
@@ -43,7 +43,7 @@ using namespace std;
     phys_t timeStep;
     phys_t restLength;
   #if TEST_CONVERGENCE
-    phys_t * averageSpeed;
+    phys_t * meanSquareNetForce;
```

Comment:
I see, okay -- I misunderstood the code. `meanSquareNetForce` is not actually a mean, it's just the sum of squares, and it's divided by the number of nodes later. Perhaps change the name to indicate this?